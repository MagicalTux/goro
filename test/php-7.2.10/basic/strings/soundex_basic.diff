     *** Testing soundex() : basic functionality ***
002- string(4) "E460"
003- string(4) "G200"
004- string(4) "H416"
005- string(4) "K530"
006- string(4) "L300"
007- string(4) "L222"
008- bool(true)
009- bool(true)
010- bool(true)
011- bool(true)
012- bool(true)
013- bool(true)
014- bool(false)
015- bool(false)
016-  
017- ===DONE===
002+ 
003+ Fatal error: Uncaught Error: Call to undefined function soundex in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/soundex_basic.php:8
004+ Stack trace:
005+ #0 {main}
006+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/soundex_basic.php on line 8
