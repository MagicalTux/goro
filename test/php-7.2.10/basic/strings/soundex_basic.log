
---- EXPECTED OUTPUT
*** Testing soundex() : basic functionality ***
string(4) "E460"
string(4) "G200"
string(4) "H416"
string(4) "K530"
string(4) "L300"
string(4) "L222"
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
bool(true)
bool(false)
bool(false)
 
===DONE===
---- ACTUAL OUTPUT
*** Testing soundex() : basic functionality ***

Fatal error: Uncaught Error: Call to undefined function soundex in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/soundex_basic.php:8
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/soundex_basic.php on line 8
---- FAILED
