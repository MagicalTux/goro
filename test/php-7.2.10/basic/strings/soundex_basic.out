*** Testing soundex() : basic functionality ***

Fatal error: Uncaught Error: Call to undefined function soundex in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/soundex_basic.php:8
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/soundex_basic.php on line 8